oops_basics/oops_basics/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System;
+using System.Collections.Generic;
+
+namespace oops_basics
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            string myTitle="book";
```

Comment:
@VarunRastogi1 -  now we dont need to initialize myTitle. Remove this initialization.